### DISCUSSION TITLE
brew install gcc giving error in macbook big sur

### DISCUSSION THREAD

**Discussion by shaknoah** [Homebrew/discussions#2342] [Casks] [answered]
shaknoah@shaknoahs-MacBook-Air ~ % brew install gcc
Warning: No available formula or cask with the name "gcc".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

**Comment by carlocab** [comment, marked as answer]
What is the output of `brew config` and `brew doctor`?

**Comment by shaknoah** [comment]
shaknoah@shaknoahs-MacBook-Air ~ % brew config
HOMEBREW_VERSION: 3.2.17-133-g890190c
ORIGIN: https://github.com/Homebrew/brew
HEAD: 890190c0f30b36512835c846be004b0253b062ad
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/brew
Core tap HEAD: 890190c0f30b36512835c846be004b0253b062ad
Core tap last commit: 2 days ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit icelake
Clang: 13.0.0 build 1300
Git: 2.30.1 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.6-x86_64
CLT: 13.0.0.0.1.1630607135
Xcode: 13.0




shaknoah@shaknoahs-MacBook-Air ~ % brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Homebrew/homebrew-core was not tapped properly! Run:
  rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
  brew tap homebrew/core

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9-config
  /Library/Frameworks/Python.framework/Versions/3.9/bin/python3-config

**Reply by carlocab** [reply]
Have you tried following this?
```
Warning: Homebrew/homebrew-core was not tapped properly! Run:
rm -rf "/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core"
brew tap homebrew/core
```

**Comment by shaknoah** [comment]
Thanks after running these commands i am able to install gcc .
